src/dmd/backend/cgobj.d

```diff
@@ -460,24 +460,12 @@ version (X86ASM)
 {
 int insidx(char *p,uint index)
 {
-    // https://issues.dlang.org/show_bug.cgi?id=19974
-    static if (__VERSION__ >= 2086)
-        asm
-        {
-            naked                           ;
-            mov     EAX,index - [ESP]       ;
-            mov     ECX,p - [ESP]           ;
-        }
-    else
-        asm nothrow
-        {
-            naked                           ;
-            mov     EAX,index - [ESP+4]     ;
-            mov     ECX,p - [ESP+4]         ;
-        }
```

Comment:
Not sure about this change, but I think I merged this incorrectly. See e.g. https://github.com/dlang/dmd/pull/10045

Reply:
Yeah, looks good.